### DISCUSSION TITLE
Suppressing words

### DISCUSSION THREAD

**Discussion by cindyairops** [deepgram/community#102] [General help] [answered]
Is it possible to suppress the array of words from the output? 
If I choose to enable utterances, there are both words, utterances, and words per utterance as well.

**Comment by briancbarrow** [comment]
> Is it possible to suppress the array of words from the output?

Yes. Instead of a positive number with each word, you would use a negative number. So your keywords property would look like `keywords: ["suppressed:-10", "words:-5", "here:-3"]`

Can you clarify the last part of your question?

**Reply by cindyairops** [reply]
I'm trying to see if its possible to suppress this output: 
<img width="213" alt="Screen Shot 2023-03-21 at 1 36 18 PM" src="https://user-images.githubusercontent.com/128432813/226694697-e8544e4b-3a54-458b-8218-0e8ab6d6941d.png">

The utterances configuration also returns individual words which I would also want to suppress.

**Reply by jpvajda** [reply, marked as answer]
@cindyairops  I think checking out our Docs on [Keyword Suppression](https://developers.deepgram.com/docs/keywords#suppress-a-keyword) might help you here.  

To use keyword suppression, send one or more instances of the keywords parameter in the query string and append a negative intensifier, which will suppress the recognition of the specified word: 

For example:

`keywords=as:-10`
`keywords=it's:-10`
`keywords=an:-10`
